lxqt-leave/main.cpp

```diff
@@ -106,6 +106,6 @@ int main(int argc, char *argv[])
                 Qt::AlignCenter,
                 dialog.size(),
                 qApp->desktop()->screenGeometry(QCursor::pos())));
-    dialog.setMaximumSize(dialog.minimumSize());
+    dialog.setMaximumSize(dialog.size());
```

Comment:
Wouldn't it be better to use `sizeHint()`? Does `size()` return reliable size if the widget wasn't ever shown?